release-calendar/src/client/components/ChannelRow.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+
+export interface ChannelRowProps {
+  channel: string;
+  RTV: string;
+  isDisplayed: boolean;
+}
+
+export class ChannelRow extends React.Component<ChannelRowProps, {}> {
+  render(): JSX.Element {
+    const channel = this.props.channel;
+    const RTV = this.props.RTV;
+    const enlarge = this.props.isDisplayed;
```

Comment:
There's really no need to assign these temp variables - it's okay to use the longer name directly because it's not that long and it doesn't take a longer time to access, and for each branch you only use it once